tests/network/autofs_server.pm

```diff
@@ -20,18 +23,33 @@ use warnings;
 
 sub run {
     select_console "root-console";
-    my $test_share_dir = "/tmp/nfs/server";
+    my $test_share_dir     = "/tmp/nfs/server";
+    my $nfsidmap_share_dir = "/home/tux";
     if (is_opensuse) {
         zypper_call('modifyrepo -e 1');
         zypper_call('ref');
     }
+    # autofs
     zypper_call('in nfs-kernel-server');
     assert_script_run "mkdir -p $test_share_dir";
     assert_script_run "echo It worked > $test_share_dir/file.txt";
     assert_script_run "echo $test_share_dir *(ro) >> /etc/exports";
-    assert_script_run "cat /etc/exports";
     systemctl 'start nfs-server';
     validate_script_output("systemctl --no-pager status nfs-server", sub { m/Active:\s*active/ }, 180);
+
+    # nfsidmap
+    assert_script_run "echo N > /sys/module/nfsd/parameters/nfs4_disable_idmapping";
+    zypper_call('in nfsidmap');
+    systemctl 'restart nfs-idmapd';
+    assert_script_run "nfsidmap -c";
+    assert_script_run "useradd -m tux";
```

Comment:
Do we really need to create a new user? Maybe we can use already existing user created during installation and you can get by `my $username = $testapi::username;`.

Reply:
Hmm, in this test we need both a case where the user exists on the server but not on the client, and a case where it exists on both. I assume the existing user would be the same on both, so this would be suitable here. Regardless, thank you for that information, I wasn't aware of that.